config/modules.config.php

```diff
@@ -12,6 +12,7 @@
     'Zend\Filter',
     'Zend\Hydrator',
     'Zend\InputFilter',
+    //'Zend\Mvc\Console', // activate it if you use zend-mvc v3
```

Comment:
zend-mvc-console has integration with zend-component-installer; there's no reason to add a commented-out entry, as installation of the component will add the entry.